src/ChatPage.cpp

```diff
@@ -448,15 +448,13 @@ void
 ChatPage::deleteConfigs()
 {
         QSettings settings;
+
+        QString profilePrefix = (UserSettings::instance()->profile() == "default" ? "" : QString("profile/%1").arg(UserSettings::instance()->profile()));
+        settings.beginGroup(profilePrefix);
```

Comment:
Shouldn't this be in an if instead? or does beginGroup("") work?

Reply:
Good point, I'm not sure

Reply:
Let's just go with the safe variant, aka an if then :3